Sources/EventTypeHandler.swift

```diff
@@ -44,13 +46,15 @@ struct EventTypeHandler {
             return nil
         }
 
+        let userId = UserId(userInfo: userInfo).id ?? ""
```

Comment:
See the comment below.

Reply:
Basically, the `?? ""` isn't needed if we just accept that userId is optional